client/asset/interface.go

```diff
@@ -93,7 +93,8 @@ type Wallet interface {
 	// associated fees that the wallet can support for the specified DEX. The
 	// fees are an estimate based on current network conditions, and will be <=
 	// the fees associated with the Asset.MaxFeeRate. For quote assets, lotSize
-	// will be an estimate based on current market conditions.
+	// will be an estimate based on current market conditions. lotSize must
+	// not be zero and will cause a panic if so.
```

Comment:
"will cause a panic if so" wording is awkward in an interface definition because it seems to instruct backend developers to have the backend panic instead of return error, but I think this comment was noting the observed behavior of the current backends, and the comment was meant as a caution to consumers.  I don't see a problem with an asset backend developer checking for zero lot size and returning an error rather than panic.

Reply:
Ok.

Reply:
Hopefully did that right.

Reply:
I might have miscommunicated a bit.  "don't see a problem with an asset backend developer checking for zero lot size and returning an error rather than panic". So I just meant the interface comment saying it will panic was out of place.  I have absolutely no objection to the `if lotSize == 0 {` in maxOrder.  That was good.  Just that interface comments are fundamentally different from implementation comments in their purpose.

Reply:
> Hopefully did that right.

Yah, last commit looks great